YoutubeDownloader/YoutubeDownloader.csproj

```diff
@@ -1,28 +1,47 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <UseWPF>true</UseWPF>
-    <ApplicationIcon>../favicon.ico</ApplicationIcon>
+    <Nullable>enable</Nullable>
+    <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
```

Comment:
What is this for?

Reply:
This is part of every Avalonia template, it is needed when publishing trimmed or with NativeAOT, if not doing that, it does nothing